engine/src/main/scala/cromwell/engine/workflow/lifecycle/initialization/WorkflowInitializationActor.scala

```diff
@@ -45,18 +45,27 @@ object WorkflowInitializationActor {
   case object WorkflowInitializationAbortedResponse extends EngineLifecycleActorAbortedResponse with WorkflowInitializationResponse
   final case class WorkflowInitializationFailedResponse(reasons: Seq[Throwable]) extends WorkflowLifecycleFailureResponse with WorkflowInitializationResponse
 
-  def props(workflowId: WorkflowId,
+  def props(workflowIdForLogging: WorkflowId,
+            rootWorkflowIdForLogging: WorkflowId,
             workflowDescriptor: EngineWorkflowDescriptor,
             ioActor: ActorRef,
             serviceRegistryActor: ActorRef,
             restarting: Boolean): Props = {
-    Props(new WorkflowInitializationActor(workflowId, workflowDescriptor, ioActor, serviceRegistryActor, restarting)).withDispatcher(EngineDispatcher)
+    Props(new WorkflowInitializationActor(
+      workflowIdForLogging,
+      rootWorkflowIdForLogging,
```

Comment:
TOL hmm now I'm almost wondering if they should be wrapped in distinct "root" and "possibly not root" types. Up to you...